plugins/kotlin/j2k/shared/tests/testData/newJ2k/postProcessing/ConvertConcatenationToRawString.java

```diff
@@ -1,15 +1,21 @@
 // IGNORE_K2
 public class Test {
-    String s1 = "asdf\n" +
-               "asdfasdf";
-    String s2 = "asdf\n" +
-               "asdfasdf\n";
-    String s3 = "asdf\n" +
-               "nadfadsf\n" +
-               "asdfasdf";
-    String s4 = "asdf\n" +
-               "nadfadsf\n" +
-               "asdfasdf\n";
+    String s1 = """
```

Comment:
Now you are converting Java multiline strings to Kotlin raw strings.

1. The test data name should be changed.
2. It seems like now this test mostly duplicates `testData/newJ2k/newJavaFeatures/textBlocks". Please migrate important cases there or remove this test entirely.

Reply:
Actually, it makes more sense to leave the original test data to assert that Java string concatenations are left as concatenations.